contracts/gateway.fc

```diff
@@ -18,21 +18,29 @@ const size::call_data::max = 2048 * 8; ;; 2 Kilobytes of call data
 const op::internal::donate = 100;
 const op::internal::deposit = 101;
 const op::internal::deposit_and_call = 102;
+const op::internal::call = 103;
 
+;; tss ops
 const op::external::withdraw = 200;
+const op::external::increase_seqno = 205;
 
+;; authority ops
 const op::authority::set_deposits_enabled = 201;
 const op::authority::update_tss = 202;
 const op::authority::update_code = 203;
 const op::authority::update_authority = 204;
+const op::authority::reset_seqno = 206;
 
 ;; GAS FEES =========================================
-;; These constants are empirically determined based on tests
-;; and slighly bumped to cover all other non-gas fees (e.g. storage, fwd_fee, etc.)
+;; These constants are EMPIRICALLY determined based on tests
```

Comment:
Why uppercase?
If this is an important statement, I would rather use a IMPORTANT prefix